UI/auth-youtube.cpp

```diff
@@ -55,6 +56,14 @@ static inline void OpenBrowser(const QString auth_uri)
 	QDesktopServices::openUrl(url);
 }
 
+static void DeleteCookies()
+{
+	if (panel_cookies) {
```

Comment:
Wouldn't this code also delete any existing Google or YouTube sign-in that people might have in their own custom browser docks?

As an example, I might have a custom browser dock set up with a _different_ YouTube account logged in (e.g. my personal account vs my "brand account"), or I might just be logged in to my personal Google account.

Using the global panel cookie manager to store the service-specific cookie information would conflate those two and mean that logging out of my brand account would also break my existing logins.

Vice-versa it would "magically" make the service docks use my personal account that I might be logged-in with in another dock.

IMO the service-specific docks need their own cookie manager (and share it if necessary) and not tap into the cookie manager used by user-managed docks.

Reply:
"Custom Browser Dock" are normally not bound to the `panel_cookie` cookie manager.

Reply:
`panel_cookie` is used by service integrations (OAuth browser and docks).

Reply:
Yes, the shared `panel_cookie` manager is only being used for non-custom (i.e. OBS native in some sense) browser docks.

Reply:
Got it, so users should be unaffected by this change. I'd still rather have services not being interfere with each others' browser dock state, but this particular change is harmless then.